packages/number-field/src/vaadin-number-field.js

```diff
@@ -299,15 +299,13 @@ export class NumberField extends InputFieldMixin(SlotStylesMixin(ThemableMixin(E
         incr = 0;
       } else if (this.max < 0) {
         value = this.max;
+        // FIXME(yuriy): find a proper solution to make correct step back
```

Comment:
Let's move this comment below, under the line where we set `value -= 2 * this.step;`

Reply:
Done.